### PR TITLE
Fix GetAndSetToMethodCallRector in class itself

### PR DESCRIPTION
**Pull request by shyim**
**Before the Fix**

Before
```php
class Enlight_View_Default
{
    protected $engine;
    public function __construct($engine)
    {
        $this->engine = $engine;
    }
}
```

After
```php
class Enlight_View_Default
{
    protected $engine;
    public function __construct($engine)
    {
        $this->addService('engine', $engine);
    }
}
```

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Great work, thanks for the test :+1:

File: tests/Rector/MagicDisclosure/GetAndSetToMethodCallRector/Fixture/fixture3.php.inc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+class Enlight_View_Default
+{
+    protected $engine;
+
+    public function __construct($engine)
+    {
+        $this->engine = $engine;
+    }
+}
+?>
```
Comment (TomasVotruba) [inline tests/Rector/MagicDisclosure/GetAndSetToMethodCallRector/Fixture/fixture3.php.inc]: If the code should remain the same after Rector run, just keep the first part. 

Remove the code after `-----`

**Comment by shyim** [comment]
@TomasVotruba Now the tests are failing or i missed something?

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Pull request merged**